tests/functional/pv/space_reclaim/test_rbd_reclaimspace_global_timeout.py

```diff
@@ -0,0 +1,218 @@
+        """
+        Create PVCs and pods
+
+        """
+        self.pool_replica = 3
+        self.sc_obj = storageclass_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            replica=self.pool_replica,
+            new_rbd_pool=True,
+        )
+        self.pvc_obj = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            access_mode=constants.ACCESS_MODE_RWO,
+            size=200,
+        )
+        self.pod_obj = pod_factory(
+            pvc=self.pvc_obj,
+        )
+
+    @polarion_id("OCS-5441")
+    @tier1
+    @skipif_external_mode
+    @skipif_managed_service
+    @skipif_hci_provider_and_client
+    def test_rbd_space_reclaim_global_timeout(self, teardown_factory):
+        """
+        Test to verify RBD space reclamation
+
+        Steps:
+        1. Create and attach RBD PVC of size 200 GiB to an app pod.
+        2. Get the used size of the RBD pool
+        3. Create two files of size 50GiB each
+        4. Delete one file
+        5. Create global config map for reclaim space job
+        6. Restart the csi-manager-controller pod
+        6. Create ReclaimSpaceJob
+        7. Validate the timeout and used size of the pool
+
+        """
+
+        fio_filename1 = "fio_file1"
+        fio_filename2 = "fio_file2"
+
+        # Fetch the used size of pool
+        cbp_name = self.sc_obj.get().get("parameters").get("pool")
+        used_size_before_io = fetch_used_size(cbp_name)
+        log.info(f"Used size before IO is {used_size_before_io}")
+
+        # Create two 50 GB file
+        for filename in [fio_filename1, fio_filename2]:
+            self.pod_obj.run_io(
+                storage_type="fs",
+                size="50G",
+                runtime=100,
+                fio_filename=filename,
+                end_fsync=1,
+            )
+            self.pod_obj.get_fio_results()
+
+        # Verify used size after IO
+        exp_used_size_after_io = used_size_before_io + (100 * self.pool_replica)
+        used_size_after_io = fetch_used_size(cbp_name, exp_used_size_after_io)
+        log.info(f"Used size after IO is {used_size_after_io}")
+
+        # Delete a file
+        file_path = get_file_path(self.pod_obj, filename)
+        self.pod_obj.exec_cmd_on_pod(
+            command=f"rm -f {file_path}", out_yaml_format=False
+        )
+
+        # Verify whether file is deleted
+        try:
+            check_file_existence(pod_obj=self.pod_obj, file_path=file_path)
+        except CommandFailed as cmdfail:
+            if "No such file or directory" not in str(cmdfail):
+                raise
+            log.info(f"Verified: File {file_path} deleted.")
+
+        # Wait for 15 seconds after deleting the file
+        time.sleep(15)
+
+        # Create Global config map with required timeout set in openshift-storage namespace
+        log.info("Creating global config timeout")
+        global_timeout, self.cm_obj = create_csi_addons_global_timeout_configmap()
```

Comment:
If the function call doesnt create object, it is returning only "False", this will result in unpacking variable error and one value can be passed to two variables. Please update the function return to return two appropriate value.